hydra-access-controls/spec/unit/access_controls_enforcement_spec.rb

```diff
@@ -1,22 +1,34 @@
 require 'spec_helper'
 
 describe Hydra::AccessControlsEnforcement do
-  before(:all) do
-    class MockController
-      include Hydra::AccessControlsEnforcement
-      attr_accessor :params
+  let(:controller) { MockController.new }
+  let(:method_chain) { MockController.search_params_logic }
+  let(:search_builder) { MockSearchBuilder.new(method_chain, controller) }
 
-      def current_ability
-        @current_ability ||= Ability.new(current_user)
-      end
+  class MockController
+#    include Blacklight::Catalog
```

Comment:
Can we remove these commented lines or provide explanation about why they're commented?